src/components/Navbar/index.tsx

```diff
@@ -4,16 +4,20 @@ import NavbarContent from './Content';
 export interface NavbarProps {
   heroImages?: { logo: string; title?: string };
   titleImages?: { light: string; dark: string };
+  isAlgolia?: boolean
+  isToggle?: boolean
```

Comment:
```suggestion
  isThemeSwitcherShown?: boolean
```

lets go with something more descriptive